campaigns/rustreach/how-I-came-to-rust.md

```diff
@@ -0,0 +1,58 @@
+
+My path is highly individual, but yours will be, too. Own that. It's great.
+
+## Picking up Rust
+
+I am certainly an early adopter of Rust. I picked it up as a hobby in the early versions, Rust 0.4 was the first version I tried out and around version 0.6, I started writing notable pieces of software in it. That was in 2013. Rust - for me - was the chance to see a language being built. I had no interested in writing a programming language, but I was curious how the process works. I'm still amazsed on how agressively languages can change when compatibilty is no issue. Back then, _all Rust code in existence_ would break _regularly_.
+
+But there were two things I found very notable:
+
+* The openness of the discussion and the ease of participation
+* The air around the community to build something to _use_
+
+Rust was always the attempt to apply techniques instead of coming up with their own (obviously, the process means that you need to come up with your own things). It was interesting to watch the process and rare to see that. I enjoyed watching and coding along.
+
+## Rust Berlin
+
+In 2014, I got in touch with @johannh about running a meetup. We were both curious about Rust, but wanted to get in touch with other people who were interested. So we formed Rust Berlin and quickly thereafter the Rust Hack & Learn, where - instead of coming and listening to a talk - you could pass by and hack on your own project. The Hack & Learn is still running every other week.
+
+We later went on to co-found RustFest with others.
+
+## Joining the Rust project
+
+There's a difference between programming Rust and being involved in the Rust project. For me, the difference is that project work somewhat goes above the sphere of your personal projects. Rust Berlin, I wouldn't consider a part of my project work. Helping other people out with my experiences, I do. Shortly after the 1.0 release, Steve contacted me whether I want to take part in the newly formed community team.
+
+I was very impressed by this. Most projects don't have an explicit community team. Sure, there's community workers, but they are rarely integrated into the project on an offical basis. Being recognised for my previous experience in that area and my opinions on a project scale had never happened to me, though. Also, the ability to think about FOSS community issues in a far wider scale than I had before was just too teasing to be true. I accepted :).
+
+## Why I'm staying
+
+I guess you won't be able to draw that much from that story. All the conditions might not apply to you. So I guess it's important to tell you why I am still around. And that's because I think the Rust project gets a lot of things _right_. Some of these things weren't always right and there's always things that need to be fixed. A previous draft of this text had included some of the things that were broken. I decided to remove them, because they are _fixed_. For me, Rust - the project - is based on a couple of things:
```

Comment:
This is a style comment which is somewhat individual / personal preference, so please feel free to stick with your preferred style! But if you like, I think that you do not need to be apologetic here :)* I think you could perhaps reduce the first couple of sentences down to something like "Your path into Rust might be different to mine, but next I'd like to tell you why I am still around..." ? 

*It has also been my tendency previously to apologise / try to reduce expectations a bit upfront (for example at the start of a talk to say "This might not apply to all of you...", "I'm not an expert on this..." etc). But I have heard good advice from others that it is better to avoid doing that, and be a bit more positive/bold! :)